src/azul/lambdas.py

```diff
@@ -97,11 +97,19 @@ class Lambdas:
     def _lambda(self):
         return aws.lambda_
 
-    def list_lambdas(self) -> list[Lambda]:
+    def list_lambdas(self, deployment: str = None) -> list[Lambda]:
+        def lambda_deployment(function: JSON) -> Optional[str]:
+            variables = function['Environment']['Variables']
+            try:
+                return variables['AZUL_DEPLOYMENT_STAGE']
+            except KeyError:
+                return None
```

Comment:
For which functions is this branch taken?

Reply:
None. I will remove this.